build_tools/benchmarks/convperf/build_and_run_convperf.sh

```diff
@@ -47,7 +47,10 @@ popd # external/iree
 popd # ${REPO_DIR}
 
 # Build ConvPerf.
-cmake -GNinja -B "${BUILD_DIR}" "${REPO_DIR}"
+cmake -GNinja \
+  -DCMAKE_C_COMPILER="${CC:-clang}" \
+  -DCMAKE_CXX_COMPILER="${CXX:-clang++}" \
```

Comment:
At the point you've set CC and CXX, it should be necessary to explicitly specify these arguments here, but it's OK if you want to.

Reply:
I'll leave it in :)